<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_garante_historial</class>
 <widget class="QWidget" name="form_garante_historial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historial Garante</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(222, 222, 222);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>1020</width>
     <height>561</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>172</red>
         <green>55</green>
         <blue>26</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>11</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 55, 26);
font:  75 11pt &quot;Century Schoolbook L&quot;;
selection-background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normalon>Íconos/silueta-masculina-de-usuario.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>Historial Garante</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>981</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Schoolbook L</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 235, 231);
font:  75 10Pt &quot;Century Schoolbook L&quot;;
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="icon">
       <iconset>
        <normalon>Íconos/calidad-de-la-pagina.png</normalon>
       </iconset>
      </attribute>
      <attribute name="title">
       <string>Garantías Existentes</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>940</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
background-color: rgb(239, 235, 231);
border-style: solid;
border-width: 5px;
border-color: rgb(172, 55, 26);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}

QTextEdit{
background-color: rgb(255, 255, 255);
}

QComboBox{
background-color: rgb(255, 255, 255);
}

QDateEdit{
background-color: rgb(255, 255, 255);
}

QPushButton{
background-color: rgb(253, 194, 179);
}

QLabel{
background-color: rgb(239, 235, 231);
}

</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTableWidget" name="tw_garantes_lista">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>901</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 235, 231);
</string>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>175</number>
        </attribute>
        <column>
         <property name="text">
          <string>Tipo Garante</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Estado credito </string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nro_credito</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Monto del Préstamo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nro. Cliente Asociado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DNI Asociado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Apellido Asociado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombre Asociado</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_51">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>51</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nro Documento:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_garante_nro_doc">
        <property name="geometry">
         <rect>
          <x>149</x>
          <y>50</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_garante_apellido">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>357</x>
          <y>23</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>42</x>
          <y>23</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_garante_nombre">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>20</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Estado :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_garante_estado">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>356</x>
          <y>56</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Century Schoolbook L</family>
          <pointsize>10</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>N°  Cliente :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_garante_nro_cliente_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>53</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Historial de Garantías </string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="boton_limpiar">
     <property name="geometry">
      <rect>
       <x>821</x>
       <y>490</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 160, 143);
font: 75 10pt &quot;KacstOne&quot;;
</string>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_guardar">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>490</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 160, 143);
font: 75 10pt &quot;KacstOne&quot;;
</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lne_garante_nro_cliente">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>17</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>526</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Schoolbook L</family>
       <pointsize>11</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>N°  Cliente :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="boton_buscar">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 160, 143);
font: 75 10pt &quot;KacstOne&quot;;</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_121">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>63</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KacstOne</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 55, 26);
font: 75 10pt &quot;KacstOne&quot;;</string>
   </property>
   <property name="text">
    <string>DNI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_garante_nro_doc_nuevo">
   <property name="geometry">
    <rect>
     <x>758</x>
     <y>61</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_estado</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>878</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_nro_cliente_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_nro_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_apellido</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_nombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_nro_doc_nuevo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>907</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_garante_nro_cliente</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
